docs/source/api/v4/jobs.rst

```diff
@@ -21,38 +21,37 @@
+	| assetUrl             | no       | Return only :term:`Content Invalidation Jobs` with this :ref:`job-asset-url`                                                         |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| cdn                  | no       | Return only :term:`Content Invalidation Jobs` for :term:`Delivery Services` within the CDN with this name                            |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| createdBy            | no       | Return only :term:`Content Invalidation Jobs` that were created by the user with this username                                       |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| deliveryService      | no       | Return only :term:`Content Invalidation Jobs` that operate on the :term:`Delivery Service` with this :ref:`ds-xmlid`                 |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| dsId                 | no       | Return only :term:`Content Invalidation Jobs` pending on the :term:`Delivery Service` identified by this integral, unique identifier |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| id                   | no       | Return only the single :term:`Content Invalidation Job` with this :ref:`job-id`                                                      |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| maxRevalDurationDays | no       | Return only :term:`Content Invalidation Jobs` with a :ref:`job-start-time` that is within the window defined by the                  |
+	|                      |          | ``maxRevalDurationDays`` :term:`Parameter` in :ref:`the-global-profile`                                                              |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
+	| userId               | no       | Return only :term:`Content Invalidation Jobs` created by the user identified by this integral, unique identifier                     |
+	+----------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------+
 
 
 .. code-block:: http
```

Comment:
Looks like this mighta been missed, but the Response Structure for `GET` has changed here. The fields returned are:

assetUrl (Remains the same)
createdBy (Remains the same)
deliveryService (Remains the same)
~keyword~ (Removed)
~parameters~ (Removed)
startTime (Remains the same)
ttlHours (Added, replaces `parameters`)
invalidationType (Added) -`REFRESH` | `REFETCH`

Also, the response example will need to be updated too.

A crude example of the returned object:
```json
{
      "id": 9,
      "assetUrl": "http://test-ds.local.tld/another/other/to/content\\.jpg",
      "createdBy": "admin",
      "deliveryService": "test-ds",
      "ttlHours": 24,
      "invalidationType": "REFRESH",
      "startTime": "2021-11-05T10:24:00.37-06:00"
  }
  ```